IBPSA/Fluid/Humidifiers/Humidifier_u.mo

```diff
@@ -107,6 +100,12 @@ equation
               {42,-28},{42,-26}},
           lineColor={255,255,255},
           fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
```

Comment:
Was this change intended?

Reply:
Yes, this change is good and consistent with the other drawing elements in this icon.